<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>450</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>746</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(255, 170, 127)F;
</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>746</width>
      <height>411</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>746</width>
      <height>450</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="Hal1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>746</width>
       <height>450</height>
      </size>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/hal1.jpg);
    background-repeat: no-repeat;
    background-position: center;
    background-color: #f0f0f0; 
}</string>
     </property>
     <widget class="QPushButton" name="bnmulai">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>210</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::StrongFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Hal2">
     <property name="maximumSize">
      <size>
       <width>746</width>
       <height>450</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/menu1.jpg);
    background-repeat: no-repeat;
    background-position: center;
    background-color: #f0f0f0; 
}</string>
     </property>
     <widget class="QPushButton" name="bnramen">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>171</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bnsushi">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>171</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bnminum">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>160</y>
        <width>171</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Hal3Ramen">
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/ramen1.jpg);
    background-repeat: no-repeat;
    background-position: center;
    background-color: #f0f0f0; 
}</string>
     </property>
     <widget class="QPushButton" name="back_2">
      <property name="geometry">
       <rect>
        <x>545</x>
        <y>363</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextconfirm_3">
      <property name="geometry">
       <rect>
        <x>642</x>
        <y>363</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelramen1">
      <property name="geometry">
       <rect>
        <x>212</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HighriseDemo</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/putihhh.jpg);
	color: rgb(220, 72, 60);
    background-repeat: no-repeat;
    background-position: center;
    background-color: transparent; 
	text-align: center;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minramen1">
      <property name="geometry">
       <rect>
        <x>234</x>
        <y>322</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plusramen1">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>322</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plusramen2">
      <property name="geometry">
       <rect>
        <x>384</x>
        <y>322</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelramen2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HighriseDemo</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/putihhh.jpg);
	color: rgb(220, 72, 60);
    background-repeat: no-repeat;
    background-position: center;
    background-color: transparent; 
	text-align: center;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minramen2">
      <property name="geometry">
       <rect>
        <x>433</x>
        <y>322</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelramen3">
      <property name="geometry">
       <rect>
        <x>609</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HighriseDemo</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/putihhh.jpg);
	color: rgb(220, 72, 60);
    background-repeat: no-repeat;
    background-position: center;
    background-color: transparent; 
	text-align: center;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minramen3">
      <property name="geometry">
       <rect>
        <x>632</x>
        <y>322</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plusramen3">
      <property name="geometry">
       <rect>
        <x>582</x>
        <y>322</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Hal4">
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/konfirmasi.jpg);
    background-repeat: no-repeat;
    background-position: center;
    background-color: #f0f0f0; 
}</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>350</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>350</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>109</y>
        <width>691</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HighriseDemo</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(202, 61, 43);</string>
      </property>
      <property name="text">
       <string>TIDAK ADA PESANAN</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Hal5">
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/proses.jpg);
    background-repeat: no-repeat;
    background-position: center;
    background-color: #f0f0f0; 
}</string>
     </property>
     <widget class="QPushButton" name="backfirst">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Hal3Sushi">
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/sushi1.jpg);
    background-repeat: no-repeat;
    background-position: center;
    background-color: #f0f0f0; 
}</string>
     </property>
     <widget class="QPushButton" name="back_3">
      <property name="geometry">
       <rect>
        <x>545</x>
        <y>363</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextconfirm_2">
      <property name="geometry">
       <rect>
        <x>642</x>
        <y>363</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plussushi1">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelsushi2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HighriseDemo</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/putihhh.jpg);
	color: rgb(220, 72, 60);
    background-repeat: no-repeat;
    background-position: center;
    background-color: transparent; 
	text-align: center;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelsushi1">
      <property name="geometry">
       <rect>
        <x>212</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HighriseDemo</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/putihhh.jpg);
	color: rgb(220, 72, 60);
    background-repeat: no-repeat;
    background-position: center;
    background-color: transparent; 
	text-align: center;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minsushi1">
      <property name="geometry">
       <rect>
        <x>234</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minsushi3">
      <property name="geometry">
       <rect>
        <x>632</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minsushi2">
      <property name="geometry">
       <rect>
        <x>433</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plussushi3">
      <property name="geometry">
       <rect>
        <x>582</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelsushi3">
      <property name="geometry">
       <rect>
        <x>609</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HighriseDemo</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/putihhh.jpg);
	color: rgb(220, 72, 60);
    background-repeat: no-repeat;
    background-position: center;
    background-color: transparent; 
	text-align: center;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plussushi2">
      <property name="geometry">
       <rect>
        <x>384</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Hal3Minum">
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/minum.jpg);
    background-repeat: no-repeat;
    background-position: center;
    background-color: #f0f0f0; 
}</string>
     </property>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>545</x>
        <y>363</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextconfirm">
      <property name="geometry">
       <rect>
        <x>642</x>
        <y>363</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plusminum3">
      <property name="geometry">
       <rect>
        <x>582</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelminum3">
      <property name="geometry">
       <rect>
        <x>609</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HighriseDemo</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/putihhh.jpg);
	color: rgb(220, 72, 60);
    background-repeat: no-repeat;
    background-position: center;
    background-color: transparent; 
	text-align: center;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minminum3">
      <property name="geometry">
       <rect>
        <x>632</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plusminum1">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelminum2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HighriseDemo</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/putihhh.jpg);
	color: rgb(220, 72, 60);
    background-repeat: no-repeat;
    background-position: center;
    background-color: transparent; 
	text-align: center;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelminum1">
      <property name="geometry">
       <rect>
        <x>212</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>HighriseDemo</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
    background-image: url(D:/Farhan File Urgency/Kuliah/Praktikum Progdas/TA/desain/putihhh.jpg);
	color: rgb(220, 72, 60);
    background-repeat: no-repeat;
    background-position: center;
    background-color: transparent; 
	text-align: center;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plusminum2">
      <property name="geometry">
       <rect>
        <x>384</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minminum1">
      <property name="geometry">
       <rect>
        <x>234</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minminum2">
      <property name="geometry">
       <rect>
        <x>433</x>
        <y>323</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
